scanner/main.go

```diff
@@ -62,6 +62,14 @@ func handleJar(path string, ra io.ReaderAt, sz int64) {
 	}
 }
 
+func trimTrailingSlashes(path string) string {
```

Comment:
I think that `path/filepath.Clean()` should do the trick here.

Reply:
Thanks for the hint